_lessons/intro-to-jinja.html

```diff
@@ -178,21 +177,31 @@ <h1 style="font-size: 300%;" class="remark-inline-code">
 {% set us = [
 {
     'name': 'me',
-    'number': 3
-},
-{
+    'number': 3,
+    'details': 'bar'
+  },
+  {
     'name': 'you',
+    'number': 100,
     'details': 'foo'
 }
 ] %}
-
+```
+--
+```jinja
 {{ us[1]['details'] }}
 ```
 --
 ```
 foo
 ```
-
+--
+```jinja
+{{ us[0].number }}
+```
+--
+```
+3
```

Comment:
**appreciation:** love the additional challenge question here to test understanding